metricbeat/scripts/config_collector.py

```diff
@@ -61,6 +70,20 @@ def collect(short=False):
     # output string so it can be concatenated
     print config_yml
 
+# Makes sure every title line is 80 chars long
```

Comment:
Actually I was going for 79 chars to leave one for the new line character. That's how it is in the master files so if this is 80 it might not align nicely.